### PR TITLE
spec: Changes to spec file

### PR DESCRIPTION
**Pull request by mkutlak**
- Removes old provides for satyr-python3
- Replaces make commands with macros

### CODE REVIEW

File: satyr.spec.in [hunk 1]
```diff
@@ -79,10 +75,10 @@ Python 3 bindings for %{name}.
...
 
-make %{?_smp_mflags}
+%make_build
 
 %install
-make install DESTDIR=%{buildroot}
+%make_install
 
 # Remove all libtool archives (*.la) from modules directory.
 find %{buildroot} -name "*.la" | xargs rm --
```
Comment (mgrabovsky) [inline satyr.spec.in]: While we're at it, this could be just `-delete` instead of the pipe.

**Comment by mgrabovsky** [comment]
I just noticed there are no xz archives for releases besides 0.26, but the spec specifically asks for `.tar.xz` in `Source0`.

**Comment by mkutlak** [comment]
> I just noticed there are no xz archives on GitHub for releases besides 0.26, but the spec specifically asks for `.tar.xz` in `Source0`.

Yep. :woman_shrugging:  I've uploaded the `0.26` for `python3-satyr` package.

The sources used to be stored on `fedorahosted.org` - https://github.com/abrt/satyr/commit/02a1b3b9ec6a9bf267fbc55d0086f3444fdb3ca9

Question is, should we stay with `.xz` (and upload sources to GH) or switch to `.gz`?

**Comment by mkutlak** [comment]
@trapas retest this please

**Comment by mkutlak** [comment]
@trapas retest this please

**Comment by mkutlak** [comment]
@trapas retest this please

**Comment by ernestask** [comment]
> Question is, should we stay with `.xz` (and upload sources to GH) or switch to `.gz`?

I imagine the sources are small enough for it to not a significant difference, so sticking with gzip would at least help avoid the manual labor.

**Comment by mkutlak** [comment]
Yeah, the difference is ~250kB.

It will require a change in downstream spec file during next release:
https://src.fedoraproject.org/rpms/satyr/blob/master/f/satyr.spec#_22

**Pull request merged**